lib/src/executable/compile_stylesheet.dart

```diff
@@ -132,7 +132,10 @@ Future<void> compileStylesheet(ExecutableOptions options, StylesheetGraph graph,
   var sourceName = source == null ? 'stdin' : p.prettyUri(p.toUri(source));
   // `destination` is guaranteed to be non-null in update and watch mode.
   var destinationName = p.prettyUri(p.toUri(destination!));
-  buffer.write('Compiled $sourceName to $destinationName.');
+  var nowStr = DateTime.now().toString();
+  // Remove fractional seconds from printed timestamp
+  var timestamp = nowStr.substring(0, nowStr.length - 7);
+  buffer.write('$timestamp - Compiled $sourceName to $destinationName.');
```

Comment:
A raw number here isn't very helpful. Can you make this something more human-readable, like ISO 8601?

Reply:
I'll note that this isn't a raw number -- `DateTime.now().toString()` gives an ISO date string.

Reply:
Oh, I suppose it's not technically ISO as it's missing the `T` between the date and time, but it is a human-readable format of `yyyy-mm-dd hh:mm:ss`.

Reply:
Thank you both for your input. @nex3 when you say raw number, what are you referring to? As @connorskees mentioned, the format for `timestamp` is `yyyy-mm-dd hh:mm:ss` (the substring method removes the last 7 characters, which include milliseconds, microseconds, and decimal point, from the printed output).